src/ts/controller/month.ts

```diff
@@ -0,0 +1,211 @@
+/**
+ * @fileoverview Controller for Month View
+ * @author NHN FE Development Lab <dl_javascript@nhn.com>
+ */
+import isUndefined from 'tui-code-snippet/type/isUndefined';
+import inArray from 'tui-code-snippet/array/inArray';
+
+import TZDate from '@src/time/date';
+import Schedule from '@src/model/schedule';
+import ScheduleViewModel from '@src/model/scheduleViewModel';
+import Collection, { Filter } from '@src/util/collection';
+import {
+  limitRenderRange,
+  getScheduleInDateRangeFilter,
+  convertToViewModel,
+  getCollisionGroup,
+  getMatrices,
+  positionViewModels
+} from '@src/controller/core';
+import { format, isSameDate, toEndOfDay, toStartOfDay } from '@src/time/datetime';
+import ModelController, { IDS_OF_DAY } from './base';
+import array from '@src/util/array';
+
+/**
+ * Filter function for find time schedule
+ * @param {ScheduleViewModel} viewModel - schedule view model
+ * @returns {boolean} whether model is time schedule?
+ */
+function _onlyTimeFilter(viewModel: ScheduleViewModel) {
+  return !viewModel.model.isAllDay && !viewModel.model.hasMultiDates;
```

Comment:
The type for 'hasMultiDates' could not be found in the schedule.

Reply:
`hasMultipleDates` property has been move from ScheduleViewModel to Schedule.